src/Modal.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import BasicModal from 'react-modal';
+
+import './Modal.css';
+
+BasicModal.setAppElement('#root')
+
+export default class Modal extends React.Component {
+  componentWillMount() {
+    this.setState({isOpen: true})
```

Comment:
You don't need this whole component. This state should go into the parent component.

Reply:
have a look at how it's implemented here: https://github.com/reactjs/react-modal

Reply:
Does it make sense to keep separate if we want to add more than 1 modal?

Reply:
It works now, and tbh this will take me ages... :/ Can we add as a refactoring tkt perhaps?